src/operations/blas3/gemm_no_local.hpp

```diff
@@ -394,76 +415,453 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, tile_type,
 
  private:
   /*!
-   * @brief Following function load a block of row_items/col_items elements from
-   * A/B matrix into reg_a/reg_b.
-   * @tparam item_size it is the size of private register: either row_items or
-   * column_item
-   * @tparam next_element : is the stride to acces the next element of A or B
-   * matrix. it is either wg_rows or wg_cols.
-   * @tparam check_block: determined whether or not the requested block is
-   * internal. false means no need to check the boundaries
-   * @tparam pointerType: the type of the input matrix
-   * @tparam check_boundary: the type of a function used for checking the
-   * boundary for blocks of data located at the edge of the input matrix
-   * @param ptr : the input matrix, either A or B.
-   * @param reg[item_size] the private array containing the input block per
-   * work-item: it is either reg_a or reg_b.
+   * @brief Loads a full block of A and multiple partial blocks of B, computing
+   * the partial result for each iteration.
+   * @tparam packet_size : Vector size
+   * @tparam check_boundary : True if block is external, false if not
+   * @tparam check_k : Whether to check in K dimension.
+   * @tparam BoundaryCheckM : Type of function for checking boundary in M
+   * dimension.
+   * @tparam BoundaryCheckN : Type of function for checking boundary in N
+   * dimension.
+   * @tparam PointerType : Type of the input pointers for A and B matrices.
+   * @param A : Input matrix A
+   * @param B : Input matrix B
+   * @param boundary_check_m : Function which checks boundary in M dimension.
+   * @param boundary_check_n : Function which checks boundary in N dimension.
+   * @param A_ptr_index : index of next element for A
+   * @param B_ptr_index : index of next element for B
+   * @param lda : leading dimension of A
+   * @param ldb : leading dimension of B
+   * @param k : the current value of K from the main loop which calls this
+   * method.
+   * @param reg_a : Pointer to private register for A.
+   * @param reg_b : Pointer to private register for B.
+   * @param reg_res : Pointer to private register for partial result.
```

Comment:
Same here, reg_res should be the result not partial

Reply:
Fixed!